### ISSUE TITLE
moment.localeData is not a function

### ISSUE THREAD

**Issue by programmeranamul** [xsoh/moment-hijri#82]
![image](https://user-images.githubusercontent.com/76878888/187337826-638d975b-891b-4f27-9257-a7864a924b05.png)

![image](https://user-images.githubusercontent.com/76878888/187337896-43059cce-c20a-470a-bb8e-8658d1b594d6.png)

I am working on a sveltekit project. When I am trying to import moment-hijri, it says to me "moment.localeData is not a function". But It's working fine in my sapper project. How can I solve this?

**Comment by mahmodHammad** [comment]
Hi @programmeranamul 

I'm facing same issue, have you managed to solve it?

**Comment by MohammedSaberMohammed** [comment]
@programmeranamul @mahmodHammad  I'm facing the same issue, any clue on how to solve this issue ?

**Comment by MiladRST** [comment]
I am also facing the same issue. Does anybody has solved it?

**Comment by programmeranamul** [comment]
Yes. I had solved the issue. I had changed some code in moment-hijri core
file.

**Comment by MiladRST** [comment]
Can you please give me an example of the changes?

**Comment by MiladRST** [comment]
based on @programmeranamul  opinion, I've changed moment-hijri core file available in node-module. I only changed moment to moment/moment.
Here is what I've done : line 12 to 22
`
/* global define */
	if (typeof define === 'function' && define.amd) {
		define(['moment/moment'], function (moment) {
			root.moment = factory(moment)
			return root.moment
		})
	} else if (typeof exports === 'object') {
		module.exports = factory(require('moment/moment'))
	} else {
		root.moment = factory(root.moment)
	}
`

**Comment by mdbaniani** [comment]
> based on @programmeranamul opinion, I've changed the moment-hijri.js core file available in node_modules/moment-hijri. I only changed moment to moment/moment. Here is what I've done : line 12 to 22 ` /* global define */
> 
> ```
> if (typeof define === 'function' && define.amd) {
> 	define(['moment/moment'], function (moment) {
> 		root.moment = factory(moment)
> 		return root.moment
> 	})
> } else if (typeof exports === 'object') {
> 	module.exports = factory(require('moment/moment'))
> } else {
> 	root.moment = factory(root.moment)
> }
> ```
> 
> `

can this be turned into a PR so that the issue can be permanently solved? it has been 2 years since the last update of this repo
